tests/v2/validation/clustertemplates/provisioning/provisioning_clustertemplate_test.go

```diff
@@ -0,0 +1,236 @@
+
+	clusterEnforcement, err := ct.client.Steve.SteveType(settings.ManagementSetting).ByID(templates.ClusterEnforcementSetting)
+	require.NoError(ct.T(), err)
+	_, err = settings.UpdateGlobalSettings(ct.client.Steve, clusterEnforcement, templates.EnabledClusterEnforcementSetting)
+	require.NoError(ct.T(), err)
+
+	verifySetting, err := ct.client.Steve.SteveType(settings.ManagementSetting).ByID(templates.ClusterEnforcementSetting)
+	require.NoError(ct.T(), err)
+	require.Equal(ct.T(), verifySetting.JSONResp["value"], templates.EnabledClusterEnforcementSetting)
+
+	log.Info("Create a standard user.")
+	user, err := users.CreateUserWithRole(ct.client, users.UserConfig(), "user")
+	require.NoError(ct.T(), err)
+	standardClient, err := ct.client.AsUser(user)
+
+	log.Info("Creating a cluster template and verifying the user is not able to create the cluster without the required permissions.")
+	clusterTemplate, err := templates.CreateRkeTemplate(ct.client, nil)
+	require.NoError(ct.T(), err)
+
+	templateRevisonConfig := mgmt.ClusterTemplateRevision{
+		ClusterConfig: &mgmt.ClusterSpecBase{RancherKubernetesEngineConfig: &mgmt.RancherKubernetesEngineConfig{
+			Network: &mgmt.NetworkConfig{Plugin: ct.provisioningConfig.CNIs[0]},
+			Version: ct.provisioningConfig.RKE1KubernetesVersions[0],
+		},
+		},
+	}
+
+	clusterTemplateRevision, err := templates.CreateRkeTemplateRevision(ct.client, templateRevisonConfig, clusterTemplate.ID)
+	require.NoError(ct.T(), err)
+
+	log.Info("Create a downstream cluster as the standard user.")
+
+	rke1Cluster := templates.UpdateClusterTemplateClusterConfig(clusterTemplate.ID, clusterTemplateRevision.ID, nil)
+
+	_, _, err = templates.CreateRke1ClusterWithTemplateAndVerify(standardClient, rke1Cluster, ct.provisioningConfig)
+	require.Error(ct.T(), err, "User is not expected to be able to create a cluster.")
+	require.Contains(ct.T(), err.Error(), "The clusterTemplateRevision is not found")
+
+	members := []mgmt.Member{
+		0: {AccessType: "owner",
+			UserPrincipalID: templates.UserPrincipalID + user.ID},
+	}
+
+	updatedMembersTemplate := *clusterTemplate
+	updatedMembersTemplate.Members = members
+
+	_, err = ct.client.Management.ClusterTemplate.Update(clusterTemplate, updatedMembersTemplate)
+	require.NoError(ct.T(), err)
+
+	log.Info("Verifying user can now create the cluster with the permissions added to the cluster template.")
+	clusterObj, clusterConfig, err := templates.CreateRke1ClusterWithTemplateAndVerify(standardClient, rke1Cluster, ct.provisioningConfig)
+	require.NoError(ct.T(), err)
+
+	log.Info("Verifying the rke1 cluster comes up active.")
+	clusterConfig.KubernetesVersion = ct.provisioningConfig.RKE1KubernetesVersions[0]
+	provisioning.VerifyRKE1Cluster(ct.T(), ct.client, clusterConfig, clusterObj)
+}
+
+func (ct *ClusterTemplateProvisioningTestSuite) TestClusterTemplateWithQuestionsAndAnswers() {
+	log.Info("Creating an rke template with questions")
```

Comment:
are questions used anywhere in here? I don't see it explicitly called, only answers.